src/main/java/prgms/vouchermanagementapp/repository/JdbcVouchers.java

```diff
@@ -0,0 +1,111 @@
+package prgms.vouchermanagementapp.repository;
+
+import org.springframework.context.annotation.Profile;
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.BeanPropertySqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
+import org.springframework.stereotype.Component;
+import prgms.vouchermanagementapp.domain.VoucherEntity;
+
+import javax.sql.DataSource;
+import java.util.*;
+
+@Component
+@Profile("release | test")
```

Comment:
이거 동작하나요..? 
```suggestion
@Profile({"release", test"})
```

이렇게 작성해주셔야할 거 같습니다. 커맨드 + P로 파라미터 힌트를 봐보세요 ㅎㅎ